timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-08 14:46:32,ERROR,asif24,create_or_pass_folder,197,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 14:46:32,ERROR,asif24,create_or_pass_folder,206,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:0387fcdd-301e-0031-1e84-b92ed1000000
Time:2024-06-08T09:16:32.6027029Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 14:46:32,INFO,asif24,home,1403,1,User asif24 logged in successfully,
2024-06-08 14:46:33,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 14:46:35,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 14:46:37,INFO,asif24,get_data_source,1993,1,table_update route successfully send data,
2024-06-08 14:46:38,INFO,asif24,Cogni_button,1710,1,CogniLink load button pressed,
2024-06-08 14:46:51,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 14:47:01,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 14:47:17,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 14:47:20,INFO,asif24,update_when_file_delete,602,1,Function update_when_file_delete Data Loaded Successfully,
2024-06-08 14:47:20,INFO,asif24,Cogni_button,1722,1,Load Data Function executed in 42.16969037055969 seconds,
2024-06-08 14:47:21,INFO,asif24,get_data_source,1993,1,table_update route successfully send data,
